apiserver/src/main/java/org/dependencytrack/resources/v1/vo/TeamsSetRequest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * This file is part of Dependency-Track.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) OWASP Foundation. All Rights Reserved.
+ */
+
+package org.dependencytrack.resources.v1.vo;
+
+import java.util.Set;
+
+import org.dependencytrack.model.validation.ValidUuidList;
+
+import jakarta.validation.constraints.NotNull;
+
+public class TeamsSetRequest {
+
+    @NotNull
+    @ValidUuidList
+    private Set<String> teams;
```

Comment:
You can use the existing `@ValidUuid` annotation here:

```suggestion
    private Set<@ValidUuid String> teams;
```